lib/NavLink.js

```diff
@@ -114,14 +114,22 @@ var NavLink = React.createClass({
                 });
             }
         }
+
+        var className = this.props.className;
+        var style = this.props.style;
+        if (isActive) {
+            className = (className ? className + ' ' : '') + (this.props.activeClass || 'active');
```

Comment:
sure, thanks! I tried to not alter the prop when not defined. Not sure how to avoid this ugly thing: `  className = (className ? className + ' ' : '') ` :-)